Normalisation and simple transformation

Morphological profiling features typically measure diverse 
cellular phenotypes such as covering intensity, shape, and texture. 
Because of the various physical measures their statistical distributions 
can show very diverse shapes (e.g. poisson, normal, negativ-binomial). 
This is at odds with feature distribution shapes from other profiling 
technologies such as gene expression, which tend to be more homogeneous 
since the quantities being measured are usually all counts, or all intensities. 
Feature transformation is therefore a critical consideration for morphological profiling. 
The standard in the field currently is to perform some type of logarithmic 
transformation, which tends to make feature distributions more symmetric 
and homoskedastic, and generally leads to superior model fits when 
correcting for spatial effects or performing downstream statistical analysis.
(Durbin et al. Bioinf,2002, Huber et al. Bioinf,2002)
A generalized logarithm has been successfully applied in recent profiling 
screens (Laufer et al. NatMeth, 2013, Fischer et al. Elife, 2015), 
which behaves as the natural logarithm for high feature values, and uses a 
shrinkage parameter to control noise, for negative or near-zero feature values. 
The shrinkage parameter can be chosen empirically, or more systematically using 
packages such as vsn (an approach that gained popularity in the microarray field,Huber et al. Bioinf,200). 
Transformation can be done using the cell-level data, or using per-well profiles.

In all profiling screens, diagnostic plots (such as plate plots or replicate scatters)
should be generated and inspected in order to assess for systematic artifacts. If 
spatial biases are present on plates, standard methods used for correction include 
B-score (Brideau et al. J Biomol Screen. 2003) and locfit(Loader, C. 1999, Springer NY).
B-score normalization implements a two-way median polish over row and columns, and 
can only be used when biological effects are not heavily confounded with 
technical biases in a column- or row-wise fashion e.g. when key controls are placed 
over a single column, and not distributed over the plate.
Locfit and related methods such as running averages, fit a smoothing surface to the 
plate using local spatial averaging, and return fitting residuals as spatial-corrected values. 
Both B-score and locfit methods are available in the Bioconductor high-throughput screening 
package cellHTS2 (Boutros et al. 2006, GenomeBiol), and are readily applicable to morphological 
profiling data. 

For experiments that are conducted over a long time period, systematic drift in feature 
values over time should also be assessed and corrected for using smoothing techniques. 
In addition, samples are mostly processed in technically defined batches such as plates 
or a number of plates at the same time. There for plate and batch effects need to be normalised for.
Normalization in morphological profiling typically refers to the scoring of feature values 
relative to controls so that the magnitude of a phenotypic effect can be assessed, 
and feature values can be put on a common scale. Normalization strategies should 
be assessed before the screen when the experiment is being designed, so biologists 
and analysts can agree on the choice and layout of relevant controls. 
The robust z-score is less affected by outliers than the standard z-score, and therefore more often useful in applied settings. 
If the expectation is that few wells on a plate will be phenotypically affected by the experimental treatment or condition, 
the z-score can simply be applied across all wells without the need for additional controls. 
When direct comparison to positive or negative controls is desirable, useful methods include Percent Of Control (POC), 
and Normalized Percent Inhibition (NPI). POC does as the name suggests, and can be performed with respect to a single control group.
NPI subtracts the positive control average and divides by the difference between the positive control and negative control averages. 
Finally, if absolute comparisons across plates must be done, normalizing against a common set of reference controls can be performed. 
In all cases, systematic differences between feature values between batches should also be assessed and corrected for.
Previously Birmingham et al. 
reviewed various considerations of normalisations control de-or independent
and concluded that no single method (e.g. percent of control, z-score, multiple of control) superseds all others.
Rather the choice of method for normalising technical biases strongly depends on the performed assay (Birmingham et al. 2009, NatMeth).
A popular normalization choice on the plate level is the robust z-score (i.e. z-score with median and MAD, Malo et al. 2006, NatBiotech). 


